pkg/volume/util/util.go

```diff
@@ -721,3 +721,21 @@ func CheckVolumeModeFilesystem(volumeSpec *volume.Spec) (bool, error) {
 	}
 	return true, nil
 }
+
+// MakeAbsolutePath convert path to absolute path according to GOOS
+func MakeAbsolutePath(goos, path string) string {
+	if goos != "windows" {
+		return "/" + path
```

Comment:
Can you also call filepath.Clean() on the path, so things like "//path" are simplified?

Reply:
fixed